src/libPMacc/include/math/vector/compile-time/twistVectorAxes.hpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2015 Heiko Burau
+ *
+ * This file is part of libPMacc.
+ *
+ * libPMacc is free software: you can redistribute it and/or modify
+ * it under the terms of either the GNU General Public License or
+ * the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * libPMacc is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License and the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * and the GNU Lesser General Public License along with libPMacc.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "math/vector/compile-time/Vector.hpp"
+
+namespace PMacc
+{
+namespace math
+{
+namespace tools
```

Comment:
Is there a reason why you add the namespace tools but not create a folder for it.
Might be we should let this method in the `math::CT` namespace because all other methods like `add, volume, min, max ,...` are also defined in the namespace `math::CT`